__tests__/app.test.js

```diff
@@ -4,6 +4,7 @@ const db = require("../db/connection");
 const seed = require("../db/seeds/seed");
 const endpoints= require("../endpoints.json")
 const { articleData, commentData, topicData, userData } = require("../db/data/test-data/index");
+const { expect } = require("@jest/globals");
```

Comment:
Looks like an unhelpful auto import from vscode here! Can safely delete this :)

Reply:
Got it. Thank you!